independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/jbang/JBangBuilderImpl.java

```diff
@@ -46,28 +49,28 @@
 
         try {
             Path target = Files.createTempDirectory("quarkus-jbang");
-            AppArtifact appArtifact = new AppArtifact("dev.jbang.user", "quarkus", null, "jar", "999-SNAPSHOT");
-            appArtifact.setPath(appClasses);
+            final ResolvedArtifactDependency appArtifact = new ResolvedArtifactDependency("dev.jbang.user", "quarkus", null,
+                    "jar", "999-SNAPSHOT", appClasses);
```

Comment:
It was present before your modification but should we get rid of the hard coded version?

Reply:
I think it's not important what this version actually is. I'll leave it for now.